src/IO/H5/File.cpp

```diff
@@ -63,6 +65,8 @@ H5File<Access_t>::H5File(std::string file_name, bool append_to_file)
     insert_header();
     close_current_object();
     insert_source_archive();
+    write_to_attribute<std::string>(file_id_, "InputSource.yaml"s,
+                                    {{input_source}});
```

Comment:
It looks like the argument type and `input_source` are both `std::string`s, so why are there braces here?